package.json

```diff
@@ -2,27 +2,33 @@
   "name": "pinejs-client-core",
   "version": "5.7.1",
   "description": "This module provides the core tools to enable easier interaction with the pinejs API.",
-  "main": "core.js",
-  "types": "core.d.ts",
+  "main": "index.js",
+  "types": "index.d.ts",
   "repository": {
```

Comment:
Can you also add a "browser" field and fix the versionist error so that I can easily test this with webpack, rollup & browserify?